app/gui2/playwright.config.ts

```diff
@@ -39,13 +39,17 @@ export default defineConfig({
   forbidOnly: !!process.env.CI,
   repeatEach: process.env.CI ? 3 : 1,
   ...(process.env.CI ? { workers: 1 } : {}),
+  timeout: 60000,
   expect: {
     timeout: 5000,
     toHaveScreenshot: { threshold: 0 },
   },
+  reporter: 'html',
   use: {
     headless: !DEBUG,
-    trace: 'on-first-retry',
+    // We have a problem with many transient failures; to investigate them, we need trace on all
+    // tests.
+    trace: 'on',
```

Comment:
Could this be `retain-on-first-failure`?

Reply:
No, because the most interesting traces are for transient failures, and those likely are successful on retry.